EncountifyAPI/Startup.cs

```diff
@@ -109,7 +117,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
-            app.UseMiddleware<StatisticsMiddleware>();
+            //app.UseMiddleware<StatisticsMiddleware>();
```

Comment:
I think it would be better to delete all comments like this one.

Reply:
I do not want to delete it, I just commented out the middleware just for now, because it was glitching.